internal/cli/atlas/commonerrors/errors.go

```diff
@@ -29,9 +29,9 @@ func Check(err error) error {
 		return nil
 	}
 
-	var atlasErr *atlas.ErrorResponse
-	if errors.As(err, &atlasErr) {
-		if atlasErr.ErrorCode == "TENANT_CLUSTER_UPDATE_UNSUPPORTED" {
+	apiError, ok := admin.AsError(err)
```

Comment:
this check is redundant? Can we check error code itself? If not I need to change sdk

Reply:
This is the response:
Error: https://cloud-dev.mongodb.com/api/atlas/v2/groups/647633bcac4c4035bcca9845/clusters/Cluster0 PATCH: HTTP 400 (Error code: "TENANT_CLUSTER_UPDATE_UNSUPPORTED") Detail: Cannot update a M0/M2/M5 cluster through the public API. Reason: Bad Request. Params: []

Here, I believe we want to return a clear error message rather than the one returned.